trillian/integration/ct_hammer/main.go

```diff
@@ -53,6 +57,26 @@ var (
 	getEntryAndProofBias  = flag.Int("get_entry_and_proof", 0, "Bias for get-entry-and-proof operations")
 )
 
+func newPool(servers string, cfg ctfe.LogConfig) (integration.ClientPool, error) {
```

Comment:
This function should live somewhere more accessible and be used in `RunCTIntegrationForLog` too -- maybe as `NewRandomPool()` in `ct_integration.go`.

I also wonder whether it should take `(servers, pubKeyPEMFile, prefix string)` rather than the whole `LogConfig` - wdyt?